SpokestackTrayExample/AppDelegate.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  AppDelegate.swift
+//  SpokestackTrayExample
+//
+//  Created by Cory D. Wiles on 8/7/20.
+//
+
+import UIKit
+import AVFoundation
+
+@main
```

Comment:
```suggestion
@UIApplicationMain
```
`@main` is [iOS 14+](https://developer.apple.com/documentation/swiftui/app/main())